### PR TITLE
docs(eol): updates for End-of-Life

### PR DESCRIPTION
**Pull request by Splaktar**
## PR Checklist

Please check your PR fulfills the following requirements:
- [ ] Does this PR fix a regression since 1.2.0, a security flaw, or a problem caused by a new browser version?
- [x] The commit message follows [our guidelines](https://github.com/angular/material/blob/master/.github/CONTRIBUTING.md#-commit-message-format)
- [x] Tests for the changes have been added or this is not a bug fix / enhancement
- [x] Docs have been added, updated, or were not required

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Enhancement
[x] Documentation content changes
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

- The README 
  - lists LTS details that have expired
  - doesn't mention the EOL status
  - has docs for contributing, but we aren't accepting any contributions
  - links to the forums and Gitter for official support, which is no longer available

## What is the new behavior?

- change LTS section of README to an End-of-Life section
  - remove LTS details
- add link to AngularJS EOL blog post
- remove release and contributing docs from README
  - remove Gitter and Google Groups forum links as they will no longer be supported
- update Copyright year in LICENSE
- fix broken Ionic v1 link
- remove README contributing link, forums link, and Gitter link from Getting Started page

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

Related to https://github.com/angular/angular.js/pull/17177.

cc: @mgol @mgechev @gkalpak @aaronfrost

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -10,10 +10,12 @@
  - [Developer Commits Levels](#pr_forks)
  - [Submission Guidelines](#submit)
 
-## AngularJS Material is in LTS mode
+## AngularJS Material has reached End-of-Life
```
Comment (jelbourn) [inline .github/CONTRIBUTING.md:13]: I don't think "end" should be capitalized

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -10,10 +10,12 @@
...
 
-We are no longer accepting changes that are not critical bug fixes into this project.
-See [Long Term Support](https://material.angularjs.org/latest/#long-term-support) for more detail.
+We are no longer accepting changes into this project as it has reached the end of the
+[Long Term Support](https://docs.angularjs.org/misc/version-support-status) period.
+
+Read the [end-of-life announcement](https://goo.gle/angularjs-end-of-life) for more details.
```
Comment (jelbourn) [inline .github/CONTRIBUTING.md:18]: Could we match the wording from https://github.com/angular/angular.js/pull/17177? 

Similar to that PR, I'd like to link to the actively supported thing so avoid any confusing (people are _hair-triggered_ on misinterpreting this as Angular)

(here and in other places)

File: .github/ISSUE_TEMPLATE/docs.md [hunk 3]
```diff
@@ -3,21 +3,19 @@ name: Documentation
 about: Suggest an improvement to our documentation at material.angularjs.org
 labels: "type: docs, needs triage"
 ---
-# AngularJS Material is in LTS mode
+<!--
 
-We are no longer accepting changes that are not critical bug fixes into this project.
-See [Long Term Support](https://material.angularjs.org/latest/#long-term-support) for more detail.
+---------------------------------------------------------------------
+This repo is for AngularJS Material, not Angular Material or the CDK.
```
Comment (jelbourn) [inline .github/ISSUE_TEMPLATE/docs.md:9]: nit: in official docs we try to say "Angular CDK" vs. "the CDK" (though we tend to refer to it that way in conversation)

**Comment by Splaktar** [comment]
Review feedback addressed.

**Review by jelbourn — APPROVED** [review]
LGTM

**Pull request merged**